### ISSUE TITLE
it let's me register user without providing email address. 

### ISSUE THREAD

**Issue by polinom** [szopu/django-rest-registration#9]
Here you have server where it's deployed and example request:

```
curl -X POST -d '{"username": "anyone", "password":"pass1234", "password_confirm":"pass1234"}' --header 'Content-Type: application/json' --header 'Accept: application/json' 'http://luken-qa.herokuapp.com/api/v1/accounts/register/'
```

my settings looks like this:

```
REST_REGISTRATION = {
        'REGISTER_VERIFICATION_ENABLED': True,
        "REGISTER_VERIFICATION_URL": "/verify-account/",

        'RESET_PASSWORD_VERIFICATION_URL': '/reset-password/',

        'REGISTER_EMAIL_VERIFICATION_ENABLED': True,
        'VERIFICATION_FROM_EMAIL': 'no-reply@example.com',
        "REGISTER_EMAIL_VERIFICATION_URL": "/verify-email/",
    }
```

Is this expected?

**Comment by szopu** [comment]
Thanks for reporting the bug! I was able to reproduce it with a test case. Working on fix right now...

**Comment by szopu** [comment]
Looks that badly formulated if statement was the cause. Fix will be released in next version (0.3.7).
